tests/test_behavior.py

```diff
@@ -66,9 +66,6 @@ def test_distance_behavior(
     caa1 = ak.Array(caa_p1.points, with_name="Point", behavior=behaviors)
     caa2 = ak.Array(caa_p2.points)
 
-    print(f"{ak.to_list(daa1.distance(daa2)) = }")
-    print(f"{ak.to_list(caa1.distance(caa2)) = }")
```

Comment:
This was here to diagnose the test failure.

But now it looks like the test passed, just having been run a second time. (Usually, not a good thing!)